builder/xenserver/common/config.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 	SourcePath        string        `mapstructure:"source_path"`
 
 	Firmware string `mapstructure:"firmware"`
+	SetTemplate string `mapstructure:"set_template"`
 
```

Comment:
We also need to update the iso builder [documentation file](https://github.com/ddelnano/packer-plugin-xenserver/blob/6af6320f8acde4e3f3e61f6d4cf2d11babcdfab7/docs/builders/iso/xenserver-iso.html.markdown) to include this new setting.